Library/Formula/ccnet.rb

```diff
@@ -0,0 +1,72 @@
+class Ccnet < Formula
+  desc "A framework for writing networked applications in C"
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/ccnet/archive/v4.2.8.tar.gz"
+  sha256 "a5ae0b9f4ddd07bdb3702f69da94e9a3a1c28feaa18a8b9a792aae42cec413b2"
+
+  head "https://github.com/haiwen/ccnet.git"
+
+  # FIX for homebrew autotools path
+  patch :DATA
+
+  # FIX for openssl build
```

Comment:
Can you elaborate a bit more on why the patch is needed and if this was merged upstream? Thanks.